src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -476,6 +476,43 @@ class FunSpecTest {
     assertThat(funSpec.toBuilder().build()).isEqualTo(funSpec);
   }
 
+  @Test fun receiverWithKdoc() {
+    val funSpec = FunSpec.builder("toBar")
+        .receiver(String::class, "the string to transform.")
+        .returns(String::class)
+        .addCode(CodeBlock.of("return %S", "bar"))
+        .build()
+
+    assertThat(funSpec.toString()).isEqualTo("""
+      |/**
+      | * @receiver the string to transform.
+      | */
+      |fun kotlin.String.toBar(): kotlin.String = "bar"""".trimMargin())
```

Comment:
nit: Having `""".trimMargin()` is really confusing, can you move it to the new line?